### PR TITLE
Add tabs to build instructions and minor fixes

### LINKED ISSUE #7: Build instructions improvements
I just went through the build instructions. Some thoughts:

* It would be nice if there were OS-specific build prerequisites, e.g. an `apt` command for Ubuntu. Tabs would be perfect for this
* I think "In the photonvision-server directory (${PROJECT_DIR}/photon-server):" could be shortened to "In the `photon-server/` directory" and it would be less confusing to newbies. I don't think there's any reason to specify an envvar
* The gradlew build command should probably be `./gradlew` not `gradlew` (unless that doesn't work on Windows, then both should be listed with tabs). Otherwise a Linux user unfamiliar with gradle might think they have to install it or something
* The client app is missing the `vue-cli-service` dependency. I had to manually `npm install @vue/cli-service` in the client directory to get `gradlew` to build
* Why does `./gradlew run` seem to hang at "75% EXECUTING" while running? Is this a me-issue?

### PR DESCRIPTION
**Pull request by vatanaksoytezer**
Fixes #7

### CODE REVIEW

File: source/docs/developers/build-instructions.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ Prerequisites
 ~~~~~~~~~~~~~
 
 | **Java Development Kit:** This project requires Java Development Kit (JDK) 11 to be compiled. This is the same Java version that comes with WPILib. If you don't have this JDK with WPILib, you can follow the instructions to install JDK 11 for your platform `here <https://bell-sw.com/pages/liberica_install_guide-11.0.7//>`_.
-| **Node JS:** The UI is written in Node JS. To compile the UI, Node 10 or newer is required. To install Node JS follow the instructions for your platform `the official Node JS website <https://nodejs.org/en/download/>`_.
+| **Node JS:** The UI is written in Node JS. To compile the UI, Node 10 or newer is required. To install Node JS follow the instructions for your platform `in the official Node JS website <https://nodejs.org/en/download/>`_.
```
Comment (mcm001) [inline source/docs/developers/build-instructions.rst:13]: Maybe "on" is a better word than "in"?

File: source/docs/developers/build-instructions.rst [hunk 2]
```diff
@@ -41,23 +41,53 @@ In the photonvision-client directory (${PROJECT_DIR}/photon-client):
...
+In the photon-server directory:
 
-.. code-block:: bash
+.. tabs::
+
+   .. group-tab:: Linux
+
+      ``./gradlew buildAndCopyUI``
+
+   .. group-tab:: Mac OSX
```
Comment (prateekma) [inline source/docs/developers/build-instructions.rst:52]: macOS is now the name of the OS.

File: source/docs/developers/build-instructions.rst [hunk 3]
```diff
@@ -41,23 +41,53 @@ In the photonvision-client directory (${PROJECT_DIR}/photon-client):
...
-      gradlew run
+   .. group-tab:: Linux
 
-Running the following command under the photonvision-server directory (${PROJECT_DIR}/photon-server) will build the jar under ${PROJECT_DIR}/photon-server/build/libs:
+      ``./gradlew run``
 
-.. code-block:: bash
-   
-      gradlew shadowJar 
+   .. group-tab:: Mac OSX
```
Comment (prateekma) [inline source/docs/developers/build-instructions.rst:71]: Same as above

File: source/docs/developers/build-instructions.rst [hunk 4]
```diff
@@ -41,23 +41,53 @@ In the photonvision-client directory (${PROJECT_DIR}/photon-client):
...
+
+Running the following command under the photon-server directory will build the jar under photon-server/build/libs:
+
+.. tabs::
+
+   .. group-tab:: Linux
+
+      ``./gradlew shadowJar``
+
+   .. group-tab:: Mac OSX
```
Comment (prateekma) [inline source/docs/developers/build-instructions.rst:87]: Same as above

**Pull request merged**